public/app/features/alerting/unified/components/rule-editor/DashboardPicker.test.tsx

```diff
@@ -21,40 +21,41 @@ jest.mock('react-virtualized-auto-sizer', () => {
 
 const server = setupMswServer();
 
-mockDashboardApi(server).search([
-  mockDashboardSearchItem({ uid: 'dash-1', type: DashboardSearchItemType.DashDB, title: 'Dashboard 1' }),
-  mockDashboardSearchItem({ uid: 'dash-2', type: DashboardSearchItemType.DashDB, title: 'Dashboard 2' }),
-  mockDashboardSearchItem({ uid: 'dash-3', type: DashboardSearchItemType.DashDB, title: 'Dashboard 3' }),
-]);
-
-mockDashboardApi(server).dashboard(
-  mockDashboardDto({
-    uid: 'dash-2',
-    title: 'Dashboard 2',
-    panels: [
-      {
-        type: 'graph',
-      },
-      {
-        type: 'timeseries',
-      },
-      // this one is a library panel
-      {
-        type: undefined,
-        libraryPanel: {
-          name: 'my library panel',
-          uid: 'abc123',
-        },
-      },
-    ],
-  })
-);
-
 const ui = {
   dashboardButton: (name: RegExp) => byRole('button', { name }),
 };
 
 describe('DashboardPicker', () => {
+  beforeEach(() => {
```

Comment:
The server sets up its handlers in a beforeEach, so if you want to add to it like this, you need to do it in a beforeEach after the server's setup has run. Ideally, you wouldn't be tweaking the behaviour too often anyway